epmon/config.yaml

```diff
@@ -16,6 +16,16 @@ elements:
   antiddos_skip_bad_type:
     service_type: AntiDDoS
     urls: []
+  aomv1:
```

Comment:
please add these elements also in

https://github.com/opentelekomcloud-infra/stackmon-config/blob/sd-fix/config.yaml
in each env you want to see them available